src/infrastructure/db/repositories/account.repository.ts

```diff
@@ -135,7 +135,7 @@ export class AccountRepository extends MongoRepository<UserSchema> {
         `[AccountRepository] [updateAccountPassword] - x-request-id: ${requestId}, email: ${email}`,
       );
 
```

Comment:
Maybe here you can use the user dto to avoid in the update-password.service the transformation of _id:
```suggestion
const user = await this.updatePasswordByEmail(email, password, true);

const userProfile: UserProfileDto = plainToClass(UserProfileDto, user, {
        excludeExtraneousValues: true,
      });

return userProfile;
```